app/controllers/holds_controller.rb

```diff
@@ -167,14 +167,21 @@ def past_date
     end
 
     def hold_lookup(hold_key)
-      hold_record = symphony_client.get_hold_info hold_key, current_user.session_token
-      parsed_hold = JSON.parse hold_record.body
+      parsed_hold = SymphonyClientParser::parsed_response(symphony_client,
+                                                          :get_hold_info, hold_key, current_user.session_token)
+
+      start = DateTime.now
+      while parsed_hold.dig('fields', 'item').nil? && start + 5.seconds > DateTime.now
```

Comment:
Hmm, the new global delay constant doesn't seem to get called here. I think I expected that it would be. Is there a reason it isn't?